es/cheatsheet-machine-learning-tips-and-tricks.md

```diff
@@ -1,257 +1,257 @@
 
 **19. Coefficient of determination ― The coefficient of determination, often noted R2 or r2, provides a measure of how well the observed outcomes are replicated by the model and is defined as follows:**
 
-&#10230;
+&#10230; Coeficiente de determinación: el coeficiente de determinación, a menudo indicado como R2 o r2, proporciona una medida de qué tan bien los resultados observados son replicados por el modelo y se define de la siguiente manera:
 
 <br>
 
 **20. Main metrics ― The following metrics are commonly used to assess the performance of regression models, by taking into account the number of variables n that they take into consideration:**
 
-&#10230;
+&#10230; Métricas principales: las siguientes métricas se utilizan comúnmente para evaluar el rendimiento de los modelos de regresión, teniendo en cuenta la cantidad de variables n que tienen en cuenta:
 
 <br>
 
 **21. where L is the likelihood and ˆσ2 is an estimate of the variance associated with each response.**
 
-&#10230;
+&#10230; donde L es la probabilidad y ˆσ2 es una estimación de la varianza asociada con cada respuesta.
 
 <br>
 
 **22. Model selection**
 
-&#10230;
+&#10230; Selección de modelo
 
 <br>
 
 **23. Vocabulary ― When selecting a model, we distinguish 3 different parts of the data that we have as follows:**
 
-&#10230;
+&#10230; Vocabulario - al seleccionar un modelo, distinguimos 3 partes diferentes de los datos que tenemos de la siguiente manera:
 
 <br>
 
 **24. [Training set, Validation set, Testing set]**
 
-&#10230;
+&#10230; [Conjunto de entrenamiento, Conjunto de validación, Conjunto de prueba]
 
 <br>
 
 **25. [Model is trained, Model is assessed, Model gives predictions]**
 
-&#10230;
+&#10230; [Modelo es entrenado, modelo es evaluado, modelo da predicciones]
 
 <br>
 
 **26. [Usually 80% of the dataset, Usually 20% of the dataset]**
 
-&#10230;
+&#10230; [Generalmente el 80% del conjunto de datos, Generalmente el 20% del conjunto de datos]
 
 <br>
 
 **27. [Also called hold-out or development set, Unseen data]**
 
-&#10230;
+&#10230; [También llamado hold-out o conjunto de desarrollo, Datos no vistos]
 
 <br>
 
 **28. Once the model has been chosen, it is trained on the entire dataset and tested on the unseen test set. These are represented in the figure below:**
 
-&#10230;
+&#10230; Una vez que se ha elegido el modelo, se entrena sobre todo el conjunto de datos y se testea sobre el conjunto de prueba no visto. Estos están representados en la figura a continuación:
 
 <br>
 
 **29. Cross-validation ― Cross-validation, also noted CV, is a method that is used to select a model that does not rely too much on the initial training set. The different types are summed up in the table below:**
 
-&#10230;
+&#10230; Validación cruzada - la validación cruzada, también denominada CV, es un método que se utiliza para seleccionar un modelo que no confíe demasiado en el conjunto de entrenamiento inicial. Los diferentes tipos se resumen en la tabla a continuación:
```

Comment:
i think is better "también denominada CV (por sus siglas en ingles)" or "validación cruzada, también denominada VC"